gordo/workflow/workflow_generator/resources/argo-workflow.yml.template

```diff
@@ -1125,14 +1093,10 @@ spec:
           cpu: 200m
 
 
-  - name: sql-and-cleanup
+  - name: workflow-cleanup
```

Comment:
Remember that now the model-builder should depend on the postgres step in the workflow (if "influx is enabled").

Reply:
Nice catch, safety third! :3rd_place_medal: